app/views/medicaid/documents/edit.html.erb

```diff
@@ -0,0 +1,48 @@
+<% content_for :header_title, "Success" %>
+
+<% content_for :header_javascript do %>
+  <%= javascript_include_tag "https://js.shubox.io/v1/#{Rails.application.secrets.shubox_js_id}.js", defer: true %>
+<% end %>
+
+<%= form_for @step,
+  as: :step,
+  builder: MbFormBuilder,
+  url: current_path,
+  method: :put,
+  html: { data: { 'documents-form' => 'true' } } do |f| %>
+
+  <div class="form-card" id="document-drop">
+    <header class="form-card__header">
+      <div class="form-card__title">
+        Upload your paperwork
```

Comment:
this is more of a design comment than a dev comment, but this page seems perhaps needlessly inconsistent with the docs page for SNAP flow. When moving to multi-member, we found that the gender/sex difference between SNAP and Medicaid required a lot of thinking when combining the two flows with the members table. 

In general, I think we should make them as consistent as possible when adding duplication. Not a blocker, just a comment based on yesterday's experiences.

Reply:
Could you give me a little more context?  I'm not sure I understand where the inconsistency lies between the two flows.

Reply:
if the pages were exactly the same, I'd expect we could `render template:` in the medicaid docs controller and use the same view for SNAP docs and Medicaid docs. Haven't done an exact diff here, but at the very lease the copy is different (don't remember us calling it "Paperwork" for SNAP)

Reply:
Ah - ok I see. It's a conversation we should bring up then.